.github/workflows/lint.yml

```
@@ -0,0 +1,91 @@
+# ABOUTME: GitHub Actions workflow for code quality checks and linting
+# ABOUTME: Runs formatting checks, linting, and security scans on every push and PR
+
+name: Code Quality
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)